src/test/resources/authz.json

```diff
@@ -33,12 +33,17 @@
     {
       "route": "getClientKeyById",
       "verb": "GET",
-      "roles": ["admin", "security", "m2m"]
+      "roles": ["admin", "security", "m2m", "support"]
+    },
+    {
+      "route": "getClientsWithKeys",
```

Comment:
Please pay attention, as that's not enough, you should add the test that uses this value, exactly like in other cases too.